app.js

```diff
@@ -0,0 +1,66 @@
+var app = {
+    Models: {},
+    Collections: {},
+    Views: {}
+};
+
+app.Models.TodoItem = Backbone.Model.extend({
+    defaults: {
+        // title: "",
+        completed: false
+    }
+});
+
+app.Collections.TodoItems = Backbone.Collection.extend({
+    model: app.Models.TodoItem,
+    // localStorage: new Store("backbone-todo")
+})
+
+app.Views.TodoApp = Backbone.View.extend({
+    el: ".todo",
+    initialize: function() {
+        this.newTodoField = this.$(".todo__new-todo-field");
```

Comment:
So when you are needing to select a DOM element, put a second class on it `js-new-todo` or something. These classes are scoped to the view so they can be short and sweet, and don't follow BEM.

Also, don't add these in until you need them. A bunch of these you wont need because they'll just get updated by doing a re-rendering. These can also have weird issues. If you store these in the initialize, and then a render happens, they are now references to DOM elements that have been thrown out.